src/js/main.js

```diff
@@ -73,7 +73,7 @@ function initializePrivacyModal () {
 
 function loadGoogleAnalytics () {
   // Replace "GA_MEASUREMENT_ID" with your Google Analytics Measurement ID
-  const gaMeasurementId = 'J2FCEQRZJ1'
+  const gaMeasurementId = 'G-31RN40BDBG'
```

Comment:
How do use this measurement ID? Is there login for Google Analytics? Or can we simply plug in this measurement ID and manage it at some endpoint?

Reply:
Thanks Arly for replying at: https://github.com/nathandh/mywebclass-simulation/pull/40#issuecomment-1477167499.